lib/workers/branch/schedule.ts

```diff
@@ -90,7 +90,7 @@ export function isScheduledNow(config: RenovateConfig): boolean {
   const [validSchedule, errorMessage] = hasValidSchedule(configSchedule);
   if (!validSchedule) {
     logger.warn(errorMessage);
-    return true;
+    return false;
```

Comment:
Why this change? Causes other test failures. Please revert and and open a new discussion / issue if you think this is a required change.

Reply:
I think it's a poor decision to blindly return true if the user has configured something incorrectly (you're also doing the same thing for timezones). It has most likely lead to headaches/confusing behavior in the past and will continue to do so in the future. A case I ran into was writing a test with an invalid schedule, that I expected to pass. Naturally it passed, because of this blunder. Fortunately I added further cases that didn't behave correctly. I think at the very least it should return false (since a lot of your test cases just check the "true" case), or even better, throw (and then be handled gracefully in the parent function). But I'm done arguing for logical behavior so just gonna revert this.

Reply:
We can change that logic, but we should not mix that in this pr. So best is to open a new issue to discuss the behavior change for invalid schedules.